src/nand.js

```diff
@@ -1,7 +1,7 @@
 import { and, complement } from 'ramda';
 
 /**
- * Returns true if both arguments are falsy; false otherwise.
+ * Returns false if both arguments are truesy; true otherwise.
```

Comment:
We use `truthy` instead of `truesy` to explain that something coerced to Boolean is true.
Can we pls change this working in all places of this PR ? Thank you!

Reply:
Thank you! I've pushed a fix up